kythe/java/com/google/devtools/kythe/platform/shared/KytheInlineMetadataLoader.java

```diff
@@ -0,0 +1,100 @@
+package com.google.devtools.kythe.platform.shared;
```

Comment:
This file needs a copyright comment at the top (use another Java file for the format).

Reply:
Done.